acceptance/config/pack_assets.go

```diff
@@ -0,0 +1,25 @@
+// +build acceptance
+
+package config
+
+type PackAssetManager struct {
```

Comment:
Nitpick: `Manager` in my mind entails a service class when in reality this appears to simply be a data type.

```suggestion
type PackAsset struct {
```

Reply:
Renamed throughout